.github/workflows/generic.yml

```diff
@@ -45,6 +45,7 @@ jobs:
       PY_COLORS: 1
       GS_CACHE_FILE_PATH: ".cache/genesis"
       GS_USE_NDARRAY: ${{ matrix.GS_USE_NDARRAY }}
+      GS_BETA_PURE: ${{ matrix.GS_USE_NDARRAY }} # we turn on beta pure for ndarray
```

Comment:
When on Mac, and using any gpu (vulkan or Metal), and using ndarray, this should be forced to 0, per my understanding.

I'm unclear whether other vulkan gpus have the same issue, since we've only ever tested on Mac Metal GPU, and CUDA GPU. Seems very plausible though.

Reply:
(though I dont see what kernel they are calling 🤔 )